tests/helm/helm_test.go

```diff
@@ -110,6 +110,13 @@ splunkObservability:
   tracesEnabled: false
   logsEnabled: false
 
+clusterReceiver:
+  config:
+    receivers:
+      k8s_cluster:
+        # required until breaking opencensus.resourcetype is released
+        # https://github.com/signalfx/splunk-otel-collector-chart/commit/1c06636ec4f0beff90c54773cb3e645df048261e
+        resource_attributes: null
```

Comment:
In the past we have added config converters to comply with breaking changes in default configuration. Since this was a shortlived item in the helm chart I'm not sure it's required but this should be documented in at least that project's changelog imo.